syntax = "proto3";

package shoppingService;

service Unary{
  rpc GetOrder(ClientItemList) returns (ServerItemList) {}
}

message Item{
  // numbers act as identifier, not traditional assigning
  string name = 1;
}

message ClientItemList{
  repeated Item clientItems = 1;
}

message ServerItemList{
  repeated Item serverItems = 1;
  string timestamp = 2;
}

